### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by jond55** [khkim02/test#1]
Hi, this looks like a very interesting project and approach. Do you have a pretrained model for PVANet 9 that you can share?

**Comment by sanghoon** [comment]
You can download PVANet 9 models from [pva-faster-rcnn repo.](https://github.com/sanghoon/pva-faster-rcnn)

**Comment by jond55** [comment]
Perfect, thanks!

**Comment by kingvision** [comment]
Yes, i download the model from pva-faster-rcnn repo. but i should how to use the model and how to convert caffemodel to pvanet model?
@jond55 @sanghoon @khkim02 
thanks

**Comment by zhangyunming** [comment]
is thiis the pvanet c++ test?
